src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -401,6 +355,52 @@ bool ProcessPrimaryQueue ()
 			return true;
 		}
 
+		bool ProcessMarkedPending ()
+		{
+			bool marked = false;
+			foreach (var pending in Annotations.MarkedPending ()) {
+				var assembly = GetAssemblyFromMetadataTokenProvider (pending);
+				var assemblyAction = Annotations.GetAction (assembly);
+				if (assemblyAction == AssemblyAction.Skip)
+					continue;
```

Comment:
The Initialize logic used to mark things except from Skip/Delete/CopyUsed assemblies. I think CopyUsed was a missing case, and Delete can't be passed on the command-line any more, so we only need to ignore Skip assemblies.

We aren't consistent about what happens when marking things in a Skip assembly. I'm not trying to change that here, just preserving the existing behavior - which doesn't Initialize skip assemblies, but still allows MarkStep to mark members in them - so we need a way to track marked items in Skip assemblies.

Alternatively, I could just remove the check so that Skip assemblies get initialized the same way, but I don't want to make any larger changes to Skip marking.

Reply:
I decided to get rid of the Skip check - this seems more consistent.